tests/testthat/test-distributions.R

```diff
@@ -1,20 +1,20 @@
 test_that("repeated calls rmvnorm agrees with one-shot", {
   vcv <- matrix(c(4, 2, 2, 3), ncol = 2)
   x <- runif(4)
-  set.seed(1)
-  r1 <- rmvnorm(x, vcv)
-  set.seed(1)
-  r2 <- make_rmvnorm(vcv)(x)
+  g1 <- mcstate_rng$new(seed = 42)
+  g2 <- mcstate_rng$new(seed = 42)
+  r1 <- rmvnorm(x, vcv, g1)
+  r2 <- make_rmvnorm(vcv)(x, g2)
```

Comment:
I feel like this is testing two perpendicular things, them being: `g1` and `g2` the same and `rmvnorm(arg1, arg2, arg3)` and `make_rmvnorm(arg2)(arg1, arg3)` the same but i think thats alright